### ISSUE TITLE
GET /payments/refunds and GET /order/customer-returns What is the difference between these two interfaces

### ISSUE THREAD

**Issue by sskaigit** [allegro/allegro-api#9777]
1. /payments/refunds Can retrieve the order . But the status of the order is READY_FOR_PROCESSING. The fulfillment status is NEW. 
2. /order/customer-returns Why can't this interface retrieve the order
example：07514650-635d-11ef-97df-7166871624c9

**Comment by MaciejFrackowiak** [comment]
[GET /payments/refunds](https://developer.allegro.pl/documentation/#operation/getRefundedPayments) returns a list of refunded payments. The seller makes these operations.

[GET /order/customer-returns](https://developer.allegro.pl/documentation/#operation/getCustomerReturns) returns a list of customer returns.

**Comment by sskaigit** [comment]
If the seller's operations will not change the status of the order and fulfillment. can the order continue to flow?

**Comment by MaciejFrackowiak** [comment]
Yes, a payment refund will not change the order status and fulfillment status.

**Comment by sskaigit** [comment]
Will the amount be updated on the order?
